back-end/confirmHealth.js

```diff
@@ -1,3 +1,8 @@
-const confirmHealth = (snack) => {};
+const confirmHealth = (snack) => {
+  console.log(snack)
+  if(snack.protein > 5  && snack.added_sugar < 5){
+    return true
```

Comment:
This is a good start but the actual snack health needs to check if sugar is less than 5 AND EITHER protein or fiber is greater than 4